apps/server/internal/httpapi/error_paths_test.go

```
@@ -0,0 +1,78 @@
+		{http.MethodGet, "/api/projects/" + otherID + "/agents/" + agentID, ""},
+		{http.MethodPut, "/api/projects/" + otherID + "/agents/" + agentID, agentBody},
+		{http.MethodGet, "/api/projects/" + otherID + "/issues", ""},
+		{http.MethodPost, "/api/projects/" + otherID + "/issues", issueBody},
+		{http.MethodGet, "/api/projects/" + otherID + "/issues/" + issueID, ""},
+		{http.MethodPatch, "/api/projects/" + otherID + "/issues/" + issueID, `{"status":"IN_PROGRESS"}`},
+		{http.MethodPost, "/api/projects/" + otherID + "/issues/" + issueID + "/assignment", `{"agentId":"` + agentID + `"}`},
+		{http.MethodGet, "/api/projects/" + otherID + "/runs", ""},
+		{http.MethodGet, "/api/projects/" + otherID + "/runs/" + runID, ""},
+	}
+	for _, tc := range cases {
+		req := httptest.NewRequest(tc.method, tc.path, strings.NewReader(tc.body))
```

golangci-lint (2.13.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)